components/AlbumEdit/PhotoSelectDelete.tsx

```
@@ -0,0 +1,87 @@
+import { authContext } from "@/features/auth/AuthProvider";
+import { photoSelectDelete } from "@/services/photoService";
+import { PhotosProps } from "@/types/photoTypes";
+
+export default function PhotoSelectDelete({ albumId, photos }: PhotosProps) {
+	const [selectedPhoto, setSelectedPhoto] = useState<string[]>([]);
+	const dispatch = useDispatch<AppDispatch>();
+	const { currentUser } = useContext(authContext);
+	const uid = currentUser?.uid;
+
+	console.log(photos);
+	useEffect(() => {
```

Biome (1.9.4)

[error] This hook does not specify all of its dependencies: uid

This dependency is not specified in the hook dependency list.

This dependency is not specified in the hook dependency list.


(lint/correctness/useExhaustiveDependencies)